internal/image/docker_schema2_test.go

```diff
@@ -373,30 +374,39 @@ type memoryImageDest struct {
 func (d *memoryImageDest) Reference() types.ImageReference {
 	return refImageReferenceMock{ref: d.ref}
 }
+
```

Comment:
Non-blocking: Aesthetically I like these mocks without the blank lines, but, meh, not that important. Alternatively, this could be moved, maybe as a `Forbidden*` parent, to `internal/testing/mocks`. (It currently isn’t there because there is only a single user.)